v1/random.yaml

```diff
@@ -37,7 +37,7 @@ paths:
               status: 303
               headers:
                 cache_control: '{{options.random_cache_control}}'
-                location: '../../{request.params.format}/{title_from_mobileapps.body.title}'
+                location: '../../{request.params.format}/{title_from_mobileapps.body.items[0].title}'
```

Comment:
If you'd like I can revert the change in MCS back to what it was. Now that the app is back to using summary it doesn't need to be in items[0] anymore. Up to you. Either way is fine with me.